docs/distributions/fedora/faq.md

```diff
@@ -5,3 +5,19 @@ If you already have Bootcamp installed, you might notice that the boot option fo
 # My boot hangs before getting to the installer
 
 This may be due to differences between USB-C to USB-A adapters. Try a different one if it is not working.
+
+# My touchbar is blank
+
+Follow the instructions in the [post-install guide](https://wiki.t2linux.org/guides/postinstall/#setting-up-the-touch-bar). If it still is not working, try updating your firmware.
```

Comment:
Try updating your firmware, that's a very confusing statement. What firmware? Better day macOS imo.